torch_points3d/applications/pointnet2.py

```diff
@@ -0,0 +1,195 @@
+import os
+import sys
+from omegaconf import DictConfig, OmegaConf
+from typing import Optional
+import logging
+
+from torch_points3d.applications.modelfactory import ModelFactory
+from torch_points3d.applications.modules.pointnet2 import *
+from torch_points3d.core.base_conv.dense import DenseFPModule
+
+from torch_points3d.applications.base_architectures.unet import UnwrappedUnetBasedModel
+
+# Must add multiscale batch
+from torch_points3d.datasets.multiscale_data import MultiScaleBatch
+
+from torch_points3d.core.common_modules.dense_modules import Conv1D
+from torch_points3d.core.common_modules.base_modules import Seq
+from .utils import extract_output_nc
+
+CUR_FILE = os.path.realpath(__file__)
+DIR_PATH = os.path.dirname(os.path.realpath(__file__))
+PATH_TO_CONFIG = os.path.join(DIR_PATH, "conf/pointnet2")
+
+log = logging.getLogger(__name__)
+
+
+def PointNet2(
```

Comment:
Eventually we should try to clean up these ModelFactory files, I think they are very messy and have a lot of duplication